### ISSUE TITLE
parameters after pre-training

### ISSUE THREAD

**Issue by Azhen08** [WentaoChen0813/SemanticPrompt#5]
Hi,
Thank you very much for providing the code.!
I have been unable to achieve the accuracy mentioned in your paper after running experiments. Could you please share the model parameters after pre-training for 800 epochs?
Thanks
